### PR TITLE
Add command to star a problem

### PR DESCRIPTION
**Pull request by wong2**
Usage: `lc star 396`

### CODE REVIEW

File: lib/commands/star.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  command: 'star <keyword>',
+  desc:    'star problem by name or index'
+};
+
+cmd.handler = function(argv) {
+  core.getProblem(argv.keyword, function(e, problem) {
+    if (e) return log.fail(e);
+    core.starProblem(problem, function(e) {
+      if (e) return log.fail(e);
+      log.info(h.prettyText(' Starred', true));
```
Comment (skygragon) [inline lib/commands/star.js]: Thanks dude, overall LGTM. Just wondering what if run 'lc star 1' twice, will it be un-starred in the end?

Comment (wong2) [inline lib/commands/star.js]: @skygragon nope, the api is idempotent

Comment (skygragon) [inline lib/commands/star.js]: OK, I think that is good enough for the moment. I guess there will be more patches to address related functions like 'unstar'. Also want to update local cache whenever is_favor is updated in problem json.

Comment (wong2) [inline lib/commands/star.js]: maybe we can chat on wechat:  `wong2333`

**Pull request merged**